src/sisl/geom/flat.py

```diff
@@ -73,3 +73,107 @@ def graphene(bond=1.42, atoms=None, orthogonal=False):
     if atoms is None:
         atoms = Atom(Z=6, R=bond * 1.01)
     return honeycomb(bond, atoms, orthogonal)
+
+def honeycomb_flake(shells: int, bond: float, atoms, vaccuum: float = 20.) -> Geometry:
+    """Hexagonal flake of a honeycomb lattice, with zig-zag edges.
+
+    Parameters
+    ----------
+    shells:
+        Number of shells in the flake. 0 means a single hexagon, and subsequent
+        shells add hexagon units surrounding the previous shell.
+    bond:
+        bond length between atoms (*not* lattice constant)
+    atoms:
+        the atom (or atoms) that the honeycomb lattice consists of
+    vaccuum:
```

Comment:
Spell, and all the way through

Reply:
Sorry, was on the phone, `vacuum`, and everywhere else :)

Reply:
Fixed it :+1: